### PR TITLE
Visual tweaks intro screens

### PR DESCRIPTION
**Pull request by cecileboucheron**

### CODE REVIEW

File: shared/login/forms/index.native.js [hunk 1]
```diff
@@ -62,31 +56,32 @@ const stylesLoginForm = {
 }
 const stylesHeader = {
   color: globalColors.orange,
-  marginTop: 27,
+  marginTop: globalMargins.small,
 }
 const stylesMessage = {
```
Comment (chromakode) [inline shared/login/forms/index.native.js]: `eslint` is failing, saying that this is no longer used.

**Comment by keybase-ci-visdiff** [comment]
The commits cc208b908fec...03419fcdd9a4 introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Icon-Icon_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1493244757113-cc208b908fec-03419fcdd9a4-linux/Icon-Icon_Image-cc208b908fec.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1493244757113-cc208b908fec-03419fcdd9a4-linux/Icon-Icon_Image-03419fcdd9a4.png)  
   Icon-Icon_Image comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits cc208b908fec...a67ba8dfa0ec introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Icon-Icon_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1493245415289-cc208b908fec-a67ba8dfa0ec-linux/Icon-Icon_Image-cc208b908fec.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1493245415289-cc208b908fec-a67ba8dfa0ec-linux/Icon-Icon_Image-a67ba8dfa0ec.png)  
   Icon-Icon_Image comparison
</summary>

**Comment by cecileboucheron** [comment]
@chromakode Good?

**Pull request merged**